project_euler/problem_54.py

```diff
@@ -0,0 +1,79 @@
+import re
+
+file = open('resource/poker.txt', 'r')
+
+colors = ['C', 'D', 'S', 'H']
+cards = ['2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K', 'A']
+
+
+def evaluate(hand):
+    if 'A' and 'K' and 'Q' and 'J' and 'T' in hand and check_flush(hand):
```

Comment:
This tests `"A"`, `"K"`, etc. for truth value, which is always `True`. You need to do `"A" in hand and "K" in hand and ...` or the like.